content/no/es6.md

```diff
@@ -44,65 +43,59 @@ mest sannsynlig ekvivalent med tilstanden til
 
- * [New String methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object)
+*   [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals) (shorthand properties and methods)
 
- * [Symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol)
+*   [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
 
- * [Template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings)
+*   [New String methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object)
 
-Du kan se en mer detaljert liste med sammenligninger opp mot andre motorer på
-prosjektsiden til (https://kangax.github.io/compat-table/es6/)[compat-table].
+*   [Symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol)
 
-## Hvilke ES6-funksjoner ligger i --es_staging-flagget?
+*   [Template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings)
 
- * [Classes](https://github.com/lukehoban/es6features#classes) (strict mode only)
+Du kan se en mer detaljert liste med sammenligninger opp mot andre motorer på prosjektsiden til [compat-table](https://kangax.github.io/compat-table/es6/).
 
- * [Object literal extensions(https://github.com/lukehoban/es6features#enhanced-object-literals)]
+## Hvilke ES6-funksjoner ligger i --es_staging-flagget?
 
- * [`Symbol.toStringTag`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) (user-definable results for `Object.prototype.toString`)
+*   [`Symbol.toStringTag`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) (brukerdefinert resultat av `Object.prototype.toString`, bak flagget `--harmony_tostring`)
 
-## Which ES6 features are in progress?
+## Hvilke ES6-funksjoner er under utvikling?
 
-New features are constantly being added to the V8 engine. Generally speaking, expect them to land on a future io.js release, although timing is unknown.
+Nye funksjoner legges inn i V8 motoren hele tiden. På generell basis kan man forvente at disse funksjonene blir tilgjengelig i en framtidig io.js utgivelse. Det kan ikke antas noe om hvor lang tid det tar.
 
-You may list all the *in progress* features available on each io.js release by grepping through the `--v8-options` argument. Please note that these are incomplete and possibly broken features of V8, so use them at your own risk:
+Listen over alle funksjoner som er *under utvikling* og er tilgjengelig i en io.js utgivelse kan listes ut med `--v8-options` argumentet. Vær oppmerksom på at dette er uferdige funksjoner og muligens funksjoner med feil, bruk dem på egen risiko:
```

Comment:
Isn't "bruk dem på eget ansvar" an alternative here? Isn't it more common to say "på eget ansvar" rather than "på egen risiko"?